src/Pkb/Parsobober.Pkb.Relations/Implementations/FollowsRelation.cs

```diff
@@ -34,4 +36,57 @@ public void SetFollows(TreeNode node, TreeNode parent)
                 node.LineNumber, parent.LineNumber);
         }
     }
+
+    public IEnumerable<Statement> GetFollowers<TStatement>() where TStatement : Statement
+    {
+        return _followsDictionary
+            .Where(statement => programContext.StatementsDictionary[statement.Value].IsType<TStatement>())
+            .Select(statement => programContext.StatementsDictionary[statement.Key].ToStatement());
+    }
+
+    public Statement? GetFollower(int lineNumber)
+    {
+        var followerStmt = _followsDictionary.FirstOrDefault(stmt => stmt.Value == lineNumber);
```

Comment:
Proponuje zamianę na Single zamiast First, niech się rozpierniczy jak baza posiada w sobie błąd :P
```suggestion
        var followerStmt = _followsDictionary.SingleOrDefault(stmt => stmt.Value == lineNumber);
```